sdk/communication/azure-communication-email/CHANGELOG.md

```diff
@@ -17,8 +17,6 @@
 - Custom headers in the email message are now key/value pairs.
 - The importance property was removed. Email importance can now be specified through either the `x-priority` or `x-msmail-priority` custom headers.
```

Comment:
I'd say the biggest change here is that we've changed the API to be a long running operation. We've removed the GetSendStatus API and the status can now be obtained by polling for the long running operation status.

Reply:
This is listed under the "breaking changes" section. (GitHub truncated this section since it was unchanged)